### DISCUSSION TITLE
Cake.Stylelint

### DISCUSSION THREAD

**Discussion by pascalberger** [cake-build/cake#2997] [Extension requests]
A plugin for running [stylelint CLI](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/cli.md) with the option of writing the output to a file

**Comment by gep13** [comment]
@PBRStreetGang is willing to take a stab at this one.  I am setting them up with access to the newly created repository here: https://github.com/cake-contrib/Cake.Stylelint
